src/main/resources/assets/lambdynlights/lang/de_de.json

```
@@ -4,16 +4,16 @@
   "lambdynlights.menu.canvas.1": "Mit Canvas ist es empfohlen, die dynamische Entitäten-Beleuchtung auszuschalten!",
   "lambdynlights.menu.canvas.2": "Dynamische Beleuchtung mit Canvas ist zur Zeit sehr Leistungsbedürftig!",
   "lambdynlights.option.block_entities": "Blockentitäten",
-  "lambdynlights.option.entities": "Entitäten",
+  "lambdynlights.option.entities": "Objekte",
   "lambdynlights.option.mode": "Dynamische Lichter",
-  "lambdynlights.option.water_sensitive": "Wasser-sensibel Prüfung",
+  "lambdynlights.option.water_sensitive": "Wasserempfindlichkeit",
   "lambdynlights.toast.first_time": "Automatisch ausgeschaltet, prüfen Sie die Mod- oder Videoeinstellungen nach!",
   "lambdynlights.tooltip.block_entities": "Schaltet Blockentitäten dynamische Beleuchtung ein.",
-  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verfahren der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
-  "lambdynlights.tooltip.entities": "Aktiviert die dynamische Beleuchtung von Entitäten. (Spielerbeleuchtung ist immer aktiviert)",
+  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verhalten der dynamische Beleuchtung des Creepers\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explosierende Creepers.\n- %s: steigende Beleuchtung durch explosierende Creepers.",
```

```suggestion
  "lambdynlights.tooltip.creeper_lighting": "Ändert das Verhalten der dynamische Beleuchtung bei Creepern\n- %s schaltet dieses aus.\n- %s: konstante Beleuchtung durch explodierende Creepers.\n- %s: Zunehmende Beleuchtung durch explodierende Creepers.",
```

1. _"des Creepers" -> "bei Creepern"_. The sentence applies to all creepers, not only one. Also sounds better.
2. "Explodieren" is written with a **"d"**: https://www.duden.de/suchen/dudenonline/explodierende